Repo-Integration/Self-Managed/README.md

```diff
@@ -67,24 +67,66 @@ Execution instructions:
 
 - Clone the repository & give setup.sh permissions to run
 
-```git clone https://github.com/mend-toolkit/mend-examples.git && cd mend-examples/Repo-Integration/Self-Managed && chmod +x setup.sh```
-- Add your activation key as an environment variable which will be copied to the .env file which is created by setup.sh
+```shell
+git clone https://github.com/mend-toolkit/mend-examples.git 
+cd mend-examples/Repo-Integration/Self-Managed 
+chmod +x setup.sh
+```
+- Add the following environment variables which will be copied to the .env file that is created by setup.sh
+  - Activation key which is obtained from the Mend User Interface Integration page
+  - Github.com personal access token which is important for Remediate and Renovate to prevent Gitbub rate-limiting imposed on non-authenticated requests
+  - Graylog root password which is the password used to login to the Graylog platform after the intial setup
+
+```shell
+export ws_key='your activation key between single quotes'
+export github_com_token='replace-with-your-github-token-inside-single-quotes'
+export graylog_root_password='the password you would like to use to login to graylog'
+```
 
-```export ws_key='your activation key between single quotes'```
-
-- Add your Github.com access token. This is important for Remediate and Renovate to prevent Gitbub rate-limiting imposed on non-authenticated requests.
-
-`export GITHUB_COM_TOKEN='replace-with-your-github-token-inside-single-quotes'`
-  
 - Run the setup.sh script for your appropriate source control management system as shown in options above
 
-- Run docker compose depending on how it was installed. Options defined -
-  - SCA only  - docker-compose.yaml
-  - SCA and SAST - docker-compose-sast.yaml
-    - **Note: this is currently only supported for GHE** [(a dedicated SAST scanner container)](https://docs.mend.io/bundle/integrations/page/deploy_with_docker.html#Target-Machine:-Run-the-Containers).
+- Run the following commands to increase your memory map count for graylog's elasticsearch.
 
-```docker compose -f <compose file> up```
+```shell
+sudo sh -c 'echo "vm.max_map_count=262144" >> /etc/sysctl.conf'
+sudo sysctl -p
+```
 
-- Run docker compose in detached mode depending on how it was installed.
+**NOTE: Please remove the following lines from the docker-compose file if you are running on Windows with Docker Desktop**
```

Comment:
Remove this section.  We do not want people running on Docker Desktop and this setup and script are not supported for docker desktop.

Reply:
Done.